content/tracing/setup/python.md

```diff
@@ -94,10 +94,16 @@ The ddtrace library includes support for the following libraries:
 | [httplib](https://docs.python.org/2/library/httplib.html)            | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#httplib  |
 | [Requests](http://docs.python-requests.org/en/master/)               | Fully Supported | http://pypi.datadoghq.com/trace/docs/other_integrations.html#requests |
 
+## OpenTracing
```

Comment:
I don't think we should have this section here, unless we add it back to each of the languages. 

Voting against it to reduce fragmentation. I think this messaging should go into the Advanced Usage section at the top of the Python tab.